wordpress-mobile/WordPress-iOS
# Sync the Site Name screen's title with the vertical.

Closes #18317

## To test:

Prerequisites: Access to the Site Intent and Site Name treatment groups. [See this PR for more information](https://github.com/wordpress-mobile/WordPress-iOS/pull/18298).

### Skipping the Site Intent view
1. Start Site Creation flow
2. On the Site Intent view (What's your website about?), tap "Skip"
3. Expect that the Site Name view's title is "Give your website a name"

<img width="365" alt="image" src="https://user-images.githubusercontent.com/2092798/162291687-befc19ac-4101-4889-b09c-06ce63007c46.png">

### Selecting a non-custom vertical on the Site Intent view
1. Start Site Creation flow
2. On the Site Intent view (What's your website about?), tap a vertical
3. Expect that the Site Name view's title is "Give your [localized vertical title] website a name"
4. Expect that the localized vertical title portion is blue

<img width="365" alt="image" src="https://user-images.githubusercontent.com/2092798/162291860-66a6f20a-925b-467d-965e-82d4e27a887d.png">

### Selecting a custom vertical on the Site Intent view
1. Start Site Creation flow
2. On the Site Intent view (What's your website about?), type in the input field (e.g. Testing)
3. Tap the custom vertical
4. Expect that the Site Name view's title is "Give your [custom vertical title] website a name"
5. Expect that the custom vertical title portion is blue

<img width="365" alt="image" src="https://user-images.githubusercontent.com/2092798/162292707-636c198f-88b7-4af7-ac63-5bc870067e81.png">

### 

## Regression Notes
1. Potential unintended areas of impact
- Site Name view's title rendering

7. What I did to test those areas of impact (or what existing automated tests I relied on)
Manually tested steps above.

8. What automated tests I added (or what prevented me from doing so)
None.

PR submission checklist:
